vitality/__init__.py

```diff
@@ -57,7 +61,16 @@ def login():
                 "Poping out the the user id if found in the session.")
             session.pop('user_id', None)
             username = escape(request.form['username'])
+            if alphaPattern.search(username):
+                print("valid")
+            else:
+                raise Exception("Invalid characters")
             password = escape(request.form['password'])
+            if alphaPattern.search(password):
+                print("valid")
+            else:
+                raise Exception("Invalid characters")
```

Comment:
Create a new class which inherits Exception and raise that new class instead. 

Probably called InvalidCharactersError.